### ISSUE TITLE
Configure mvim as cli tool via the settings

### ISSUE THREAD

**Issue by greenchapter** [macvim-dev/macvim#1404]
Most of the time I would start mvim from the command line, but that is only possible if you add a softlink into `/usr/local/bin/` so I would recommend a more user friendly version.

**Described solution**

I would recommend a new check box within the settings panel which will add a softlink to `/usr/local/bin/` from the GUI.
<img width="595" alt="Screenshot 2023-05-08 at 16 30 52" src="https://github.com/macvim-dev/macvim/assets/693276/27f671c1-664c-4132-a3d7-3e58f281acf8">

**Currently way to add mvim to the cli**

1. Install macvim
2. Add a softlink `sudo ln -s /Applications/MacVim.app/Contents/bin/mvim /usr/local/bin/mvim`

**Comment by ychin** [comment]
> Most of the time I would start mvim from the command line, but that is only possible if you add a softlink into `/usr/local/bin/` so I would recommend a more user friendly version.

These days we usually just recommend adding `/Applications/MacVim.app/Contents/bin` to your path directly so you don't have to make a symlink. See [`:h macvim-PATH`](https://macvim.org/docs/gui_mac.txt.html#macvim-PATH). It has pros and cons compared to making a symlink to `/usr/local/bin/mvim` but I like the current recommendation more because it is easier and requires less permission to do.

The description already exists (see the above help link) but I admit it's a little hard to discover. It could be useful to have a button to do this for the user, but it's a little harder to add a folder to `$PATH` (since the user may have different shells they use) compared to a simple symlink, which is a simple command to call (albeit needing root permissions).

**Comment by greenchapter** [comment]
> > Most of the time I would start mvim from the command line, but that is only possible if you add a softlink into `/usr/local/bin/` so I would recommend a more user friendly version.
> 
> These days we usually just recommend adding `/Applications/MacVim.app/Contents/bin` to your path directly so you don't have to make a symlink. See [`:h macvim-PATH`](https://macvim.org/docs/gui_mac.txt.html#macvim-PATH). It has pros and cons compared to making a symlink to `/usr/local/bin/mvim` but I like the current recommendation more because it is easier and requires less permission to do.
> 

Thank you very much @ychin 🤩 good comment, I changed it directly…


> The description already exists (see the above help link) but I admit it's a little hard to discover. It could be useful to have a button to do this for the user, but it's a little harder to add a folder to `$PATH` (since the user may have different shells they use) compared to a simple symlink, which is a simple command to call (albeit needing root permissions).

I think it can be very valuable to some users to bring the possibility to set this path into the macvim UI. Currently for me the macOS app [kaleidoscope](https://kaleidoscope.app/) solved that very well.

<img width="992" alt="Screenshot 2023-05-26 at 16 19 30" src="https://github.com/macvim-dev/macvim/assets/693276/94eaec13-9110-4f7f-9d95-9390cf53e625">

**Comment by ychin** [comment]
Right. Let me think about it a little. Just need to play around with it a bit and triage when I get time to work on this.
